DANS-KNAW/dd-dans-deposit-to-dataverse
# DD-412 - Imported descriptions should preserve newlines using html elements

Fixes DD-412 

Note: @jo-pol is looking at the preserving of whitespaces in EASY. Currently easy-deposit-form removes all spacing.

# Description of changes


# How to test
1. Change  dataset.xml description field of an example bag to include a newline (with enter) and paragraph (enter twice)
2. Import the bag `dd-dans-deposit-to-dataverse import -s <path to bag> <outboxdir>`
3. Check dataset in UI
4. Also check unit tests

# Related PRs 
(Add links)
* 

# Notify
@DANS-KNAW/dataversedans
